example/test.go

```diff
@@ -0,0 +1,45 @@
+package main
```

Comment:
And that file?

Reply:
Oh wow, oops... Must've done a `git add .` at some point without realizing it.